docs/content/docs/concepts/summarizer.md

```diff
@@ -8,41 +8,40 @@ aliases:
 
 ## What do I need to know?
 
-Ideally very little. If you are building on top of our runtime and dds, then you won't need to think about the summary
-at all.
+Ideally very little.
+If you are building on top of our runtime and [distributed data structure]({{< relref "glossary.md#distributed-data-structures-ddses" >}})s (DDSes), then you won't need to think about the summary at all.
 
-When users with significant latency use your application, they may see the state of the container at the summary and
-then see new operations get applied. This can be hidden by waiting for the Container to be fully connected or you can
-allow users to see recent changes.
+When users with significant latency use your application, they may see the state of the container at the summary and then see new operations get applied.
+This can be hidden by waiting for the Container to be fully connected, or by allowing users to see recent changes.
 
 ## Summary lifecycle
 
-The lifecycle of a summary starts when the Summary Manager selects the client that will summarize the state of the container.
+The lifecycle of a summary starts when the [Summary Manager](#summary-manager) selects the client that will summarize the state of the container.
 
 1. The selected client spawns a non-user runtime (including a quorum, clientId, and container) that will generate the summary.
-2. Runtime generates summary tree (more details below).
+2. The runtime generates summary tree (more details [below](#shape-of-a-summary)).
     - The timing of the summaries is determined by a few heuristics discussed below
-3. Runtime uploads summary tree to the Fluid Service storage (Historian), which returns a handle to the data.
-4. Runtime submits a "summarize" op to the server containing that uploaded summary handle.
-5. Ordering service on server stamps and broadcasts the "summarize" op.
+3. The runtime uploads summary tree to the Fluid Service storage (Historian), which returns a handle to the data.
+4. The runtime submits a "summarize" op to the server containing that uploaded summary handle.
+5. The ordering service on server stamps and broadcasts the "summarize" op.
 6. Another service on server responds to "summarize" op.
-    - The server can reject the summary by sending a "summaryNack" op referencing the sequence number of the "summarize"
-      op.
-    - The server can accept the summary, but first it must serialize the protocol state and add it to the posted
-      summary. Then it will need to send a "summaryAck" op with the new handle to the augmented summary.
-7. Runtime watches for "summaryAck"/"summaryNack" ops, using them as input to its heuristics determining when to
-   generate summaries
+    - The server can reject the summary by sending a "summaryNack" (summary negative acknowledgement) op referencing the sequence number of the "summarize" op.
+    - The server can accept the summary, but first it must serialize the protocol state and add it to the posted summary.
```

Comment:
Hm not sure if " but first it must serialize the protocol state" is up to date. Maybe a question for @hedasilv.